test/fixtures/models/scc_accounts.yml

```diff
@@ -14,6 +15,7 @@ two:
   name: twoname
   interval: never
   organization_id: <%= ActiveRecord::FixtureSet.identify(:empty_organization) %>
+  sync_date: 2021-07-05T21:33:49
```

Comment:
Is there a reason we use a different format here than at line 9?

Reply:
Not really, I just wanted to have some variation. I don't think that it really matters.

Reply:
Fine with me :smiley: